### PR TITLE
Demo Page Redesign : first two sections 

### PR DESCRIPTION
**Pull request by natapg**
mock-up: figma.com/file/niHrIRowTbqQhDO7W5CPNf/FPJS-WEB-UI-%2B?node-id=0%3A1

### CODE REVIEW

File: src/components/demo/DemoSection/DemoSection.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  return (
+    <>
+      {isLoading ? (
+        <VisitorSectionLoading />
+      ) : (
+        visitorId && <VisitorSection visitorId={visitorId} visits={visits} currentVisit={currentVisit} />
+      )}
+    </>
+  )
```
Comment (tan-90) [inline src/components/demo/DemoSection/DemoSection.tsx:56]: Do we need this fragment?
```suggestion
  return isLoading ? (
        <VisitorSectionLoading />
      ) : (
        visitorId && <VisitorSection visitorId={visitorId} visits={visits} currentVisit={currentVisit} />
      )
```

File: src/components/demo/DemoSection/VisitorSection/VisitorSection.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          </p>
+          <a href={DOC_URL.documentationUrl} className={styles.link}>
+            See documentation →
+          </a>
+        </section>
+        <section className={styles.idSection}>
+          <div className={classNames(styles.card, { [styles.incognito]: incognito })}>{card}</div>
+          <footer className={styles.footer}>
+            {incognito
+              ? 'You are in private browsing. Your visitorID remain the same'
```
Comment (tan-90) [inline src/components/demo/DemoSection/VisitorSection/VisitorSection.tsx:95]: I think this should be "Your visitorID remains the same" or "Your visitorID remained the same".

**Review by tan-90 — APPROVED** [review]
Approving in advance with minor comments, please take a look.

File: src/components/demo/DemoSection/DemoSection.tsx (hunk 1 above)
Comment (natapg) [reply src/components/demo/DemoSection/DemoSection.tsx:56]: no, we don't, thanks!

File: src/components/demo/DemoSection/VisitorSection/VisitorSection.tsx (hunk 2 above)
Comment (natapg) [reply src/components/demo/DemoSection/VisitorSection/VisitorSection.tsx:95]: Agree, good catch. I will ask Savannah about this

**Pull request merged**